scss/detail.scss

```diff
@@ -93,5 +93,97 @@
         }
       }
     }
+  }  
+  @media only screen and (max-width: $flex-mobile) {
+    height: 667px;
+    width: 100%;
+    flex-direction: column-reverse;
+  
+    .return-button {
+      align-self: center;
+      font-size: 18px;
+      line-height: 24px;
+      margin: 20px 0 0 0;
+
+      &::before {
+        margin-right: 12px;
+      }
+    }
+
+    .cover-text-container {
+      flex-direction: column;
+      align-items: center;
+      height: 570px;
+      width: 336px;
+
+      .cover-page-badge-container {
+        margin-bottom: 21px;
+        
+        .cover-page {
+          height: 299px;
+          width: 212px;
+          margin: 38px 0 0 0;
+        }
+        
+        .badge {
+          $width: 95px;
+          width: calc(#{$width} * 0.7);
+          $height: 119px;
+          height: calc(#{$height} * 0.7);
```

Comment:
If you need to do this only once i would avoid creating variables and use the values directly like this:
```scss
width: calc(95px * 0.7);
```

Otherwise i would define this rule for every badge and change only the variables value